pytest.ini

```diff
@@ -0,0 +1,15 @@
+[pytest]
+addopts = 
+    --doctest-modules 
+    --ignore=examples
+    --ignore=docs
+
+    ; TODO: The following modules are ignored due to optional dependencies, which
```

Comment:
@Scienfitz, @AVHopp: Could not find any better solution. Do you have an idea, perhaps?

Reply:
I take this as a "no" and will resolve. If you do have an idea, please let me know. I'll merge the PR now, but we can then incorporate the changes in the next one.